# Generated by Django 2.0.2 on 2018-02-27 02:07

from django.db import migrations, models


class Migration(migrations.Migration):

    initial = True

    dependencies = [
    ]

    operations = [
        migrations.CreateModel(
            name='ExceptionTraceback',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('created_at', models.DateTimeField(auto_now_add=True, db_index=True)),
                ('user_id', models.CharField(blank=True, db_index=True, default=None, max_length=255)),
                ('view_name', models.CharField(blank=True, default=None, max_length=255)),
                ('request_path', models.CharField(blank=True, default=None, max_length=1024)),
                ('exception_type', models.CharField(blank=True, default=None, max_length=255)),
                ('exception_message', models.TextField(blank=True, default=None)),
                ('traceback_log', models.TextField(blank=True, default=None)),
                ('post_data', models.TextField(blank=True, default=None)),
            ],
        ),
    ]
